packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -129,23 +127,9 @@ export const makeScheduler = async (
       return;
     }
 
-    if (
-      TimeMath.compareAbs(now, schedule.startTime) >= 0 &&
-      TimeMath.compareAbs(now, schedule.endTime) <= 0
-    ) {
-      if (auctionState !== AuctionState.ACTIVE) {
-        auctionState = AuctionState.ACTIVE;
-        auctionDriver.startRound();
-      } else {
-        auctionDriver.reducePriceAndTrade();
-      }
-    }
-
-    if (TimeMath.compareAbs(now, schedule.endTime) >= 0) {
+    const finishAuctionRound = () => {
```

Comment:
helpful to define this function. 

nit: have it return the promise for the timer cancellation, in case something wants to learn of rejection.
```suggestion
    /** @type {() => Promise<void>} */
    const finishAuctionRound = () => {
```